features/keychain/module/crypto/secp256k1.js

```diff
@@ -12,11 +15,16 @@ export const create = ({ getPrivateHDKey }) => {
   const createInstance = () => ({
     signBuffer: async ({ seedId, keyId, data, ecOptions, enc = 'der' }) => {
       assert(['der', 'raw'].includes(enc), 'signBuffer: invalid enc')
-      assert(validEcOptions(ecOptions), 'signBuffer: invalid EC option')
+      assert(isValidEcOptions(ecOptions), 'signBuffer: invalid EC option')
       const { privateKey } = getPrivateHDKey({ seedId, keyId })
       const signature = curve.sign(data, privateKey, pick(ecOptions, ['canonical']))
       return enc === 'der' ? Buffer.from(signature.toDER()) : { ...signature }
     },
+    signSchnorr: async ({ seedId, keyId, data, tweak, extraEntropy }) => {
+      const hdkey = getPrivateHDKey({ seedId, keyId })
+      const privateKey = tweak ? tweakPrivateKey({ hdkey, tweak }) : hdkey.privateKey
+      return ecc.signSchnorr(data, privateKey, extraEntropy)
+    },
```

Comment:
no need to do anything right now but I find having `signBuffer` next to `signSchnorr` a little weird. Both are signing a buffer, they are just using different signature schemes. Maybe `signBuffer` should be come `signEcdsa`?

Reply:
That would be one way to. However note that sodium also has multiple sign functions: https://github.com/ExodusMovement/exodus-oss/blob/82e5cf2e1b9334af150bf2a8bba4abbc0f74c764/features/keychain/module/crypto/sodium.js#L33